lib/daos/ridesDao.js

```diff
@@ -0,0 +1,50 @@
+import { badImplementation } from '@utils/responseInterceptors';
+import moment from 'moment';
+import { TRIP_STATUS_TYPES } from '@utils/constants';
+import { models } from '@models';
+
+export const bookARide = async (pickUpLocation, dropLocation, custId) => {
+  try {
+    const ridesData = {
+      pickupLoc:pickUpLocation,
+      dropLoc: dropLocation,
+      dateOfRide: moment(moment.now()).format('YYYY-MM-DD'),
+      bookingTime: moment(moment.now()).format("YYYY-MM-DD HH:mm:ss"),
+      custId,
+      created_at: moment(moment.now()).format("YYYY-MM-DD HH:mm:ss")
```

Comment:
Why we are storing duplicate data in different columns `dateOfRide`, `bookingTime`, `created_at`

Reply:
- Date of ride is only date on which the ride is scheduled.
- Booking time we are taking the current time at which the API is called.
- Created at will also be the same.